cds/modules/records/serializers/smil.py

```diff
@@ -65,11 +65,36 @@ def format(self):
         videos_data = self._format_videos(self.record)
         return render_template('cds_records/video.smil', videos=videos_data)
 
-    @staticmethod
-    def _format_videos(record):
+    def _sort(self, subformats):
+        """Returns the subformats sorted so that the video with resolution
```

Comment:
If I am not wrong, but I can be wrong here, the first line of the comment should be just one line ending with .
Then, long descriptions are just after.
https://www.python.org/dev/peps/pep-0257/#id17
```
Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.
```

Build will fail I guess.

Reply:
You are right. I changed the multi line comment. Thank you!